tools/tests/test_osqueryd.py

```diff
@@ -136,7 +144,8 @@ def results_exists():
             self.assertTrue(os.path.exists(pth))
 
             # Only apply the mode checks to .log files.
-            if pth.find('.log') > 0:
+            # TODO: Add ACL checks for Windows logs
+            if pth.find('.log') > 0 and os.name != "nt":
```

Comment:
No it's just looking for the presence of `.log` in the given filename, I believe this is just a string search and not a regex based one.